shell/packages/sandstorm-ui-powerbox/powerbox-server.js

```diff
@@ -302,8 +369,8 @@ Meteor.publish("powerboxOptions", function (requestId, descriptorList) {
   // packed format and base64-encoded. The publish populates a pseudo-collection called
   // `powerboxOptions`. Each item has the following fields:
   //
-  //   _id: Object with two fields: `requestId` (as passed when subscribing) and `optionId` (a
-  //       unique identifier for the option).
+  //   _id: Unique identifier string.
```

Comment:
The old `optionId` was only unique within a particular query, not across queries. In fact, optionIds for singleton options are often human-readable names. Hence the need for a compound _id. Do you really need to change this?